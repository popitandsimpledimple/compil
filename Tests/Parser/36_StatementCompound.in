var a : integer;
begin
a := 1;
begin
a := 2;
a := 3;
begin
begin
begin
a := 4;
a := 5;
begin
a := 6;
a := 7;
begin
begin
a := 123;
end;
end;
end;
end;
end;
end;
end;
end.