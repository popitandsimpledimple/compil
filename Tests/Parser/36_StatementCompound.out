program
   decls
      vars
         var
            a
            type
               integer
            no expression
   block
      compounds statements
         :=
            a
            1
         compounds statements
            :=
               a
               2
            :=
               a
               3
            compounds statements
               compounds statements
                  compounds statements
                     :=
                        a
                        4
                     :=
                        a
                        5
                     compounds statements
                        :=
                           a
                           6
                        :=
                           a
                           7
                        compounds statements
                           compounds statements
                              :=
                                 a
                                 123
